### PR TITLE
Convert Conformance View to a table and Other Minor UI Changes.

### PR DESCRIPTION
**Pull request by noranda**
# Convert Conformance View to a table

* Converted conformance view from spans to a responsive table
* Added bootstrap styles to allow collapse/expand rows to work
* Added javascript to add/remove the `hidden` class in order to get around a bug where the collapsed row had a small height, throwing off the alignment
* Changed to using a tabbable mixin to assign IDs to the rows (IDs should be unique)
* Added `Case.js` which allows breaking up the Resource Types into words which improves the look of the table at smaller resolutions

#### Conformance View
Conformance View

#### Expanded Conformance View
Expanded Conformance View

# Other Minor UI Changes.

* Fixed some minor CSS errors
* Changed background color to match Bootstrap's striped table color
* Changed footer logo

#### Footer
Footer

### CODE REVIEW

File: app/templates/components/conformance-view.hbs [hunk 1]
```diff
@@ -1,91 +1,78 @@
...
-
-        {{#each r.resource as |res|}}
-          <div class="panel">
-            <div class="panel-heading" role="tab" id="headingTest1">
-              <h4 class="panel-title">
-                <a class="collapsed" data-toggle="collapse" data-parent="#accordion" aria-expanded="false" data-target="#resource{{unbound res.fhirType}}">{{res.fhirType}}</a>
-                <span class="test-circle-holder">
-                  {{#each actions as |operation|}}
+    {{!-- <div id="resource-chart" class="col-md-12"> --}}
+    {{!-- {{#each conformance.rest as |r|}} --}}
```
Comment (jtferns) [inline app/templates/components/conformance-view.hbs]: Are we saving these in the template for a reason?

**Comment by jtferns** [comment]
Hovering over the top row's results shows a clipped popover (z-index issue?):
screen shot 2015-03-25 at 7 25 16 pm

File: app/templates/components/conformance-view.hbs (hunk 1 above)
Comment (noranda) [inline app/templates/components/conformance-view.hbs]: We sure aren't! I thought I cleared all that out. I'll fix that now.

**Comment by jtferns** [comment]
Also I can't seem to expand the conformance table rows in the servers.show route (Landing Page -> Enter Server URL -> Click on 'Conformance' tab -> attempt to see search params for a resource)...

**Pull request merged**